docs/asciidoc/static/roadmap/index.asciidoc

```diff
@@ -202,6 +147,49 @@ Logstash Forwarder. We recently delivered
 http://www.elasticsearch.org/blog/logstash-forwarder-0-4-0-released/[Logstash Forwarder 0.4.0],
 which addressed many existing issues our users have been reporting.
 
+== Performance
+[float]
+=== status: ongoing; v1.5, v2.x
+
+In the 1.5 release, we significantly improved the performance of the Grok
+filter, which is used to parse text via regular expressions. Based on our
+internal benchmarks, parsing common log formats, such as Apache logs, was 2x
+faster in Logstash 1.5 compared to previous versions. We also sped up JSON
+serialization and deserialization. In future releases of Logstash, we plan to
+incorporate additional JRuby optimizations to make the code even more efficient.
+We also plan to seek community feedback in terms of prioritizing other aspects
+of performance, such as startup time, resource utilization, and pipeline
+latency.
```

Comment:
Link this issue which covers perf plans in great detail? https://github.com/elastic/logstash/issues/3499